docs/Reference/Server.md

```diff
@@ -374,6 +378,24 @@ The possible values this property may have are:
   const fastify = require('fastify')({logger: customLogger});
   ```
 
+### `createRequestLogMessage`
+<a id="create-request-log-message"></a>
+
++ Default: `false`
+
+When logging is permitted, the user can fully customize their 
```

Comment:
Text is not matching the described functionality. default is not false, but basically `v => v`

Reply:
Ok I'll be sure to correct that thanks.